client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { PerformanceMetrics } from 'calypso/performance-profiler/types/performance-metrics';
+import {
+	metricsNames,
+	metricsTresholds,
+	mapThresholdsToStatus,
+} from 'calypso/performance-profiler/utils/metrics';
+import { MetricScale } from '../metric-scale';
+import { MetricTabBar } from '../metric-tab-bar';
+import './style.scss';
+import { StatusIndicator } from '../status-indicator';
+
+export const CoreWebVitalsDisplay = ( props: PerformanceMetrics ) => {
+	const translate = useTranslate();
+	const [ activeTab, setActiveTab ] = useState< string >( 'lcp' );
+
+	const { name: metricName, displayName } = metricsNames[ activeTab as keyof typeof metricsNames ];
+	const value = props[ activeTab as keyof PerformanceMetrics ];
+	const valuation = mapThresholdsToStatus( activeTab as keyof typeof metricsTresholds, value );
+
+	const displayValuation = ( valuation: string ) => {
+		switch ( valuation ) {
+			case 'good':
+				return 'is good';
+			case 'needsImprovement':
+				return 'needs improvement';
+			case 'bad':
+				return 'is bad';
+			default:
+				return 'unknown';
+		}
+	};
+
+	const { good, needsImprovement } = metricsTresholds[ activeTab as keyof typeof metricsTresholds ];
+	const formatUnit = ( value: number ) => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return +( value / 1000 ).toFixed( 2 );
+		}
+
+		return value;
+	};
+
+	const displayUnit = () => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return 'seconds';
+		}
+
+		if ( [ 'inp' ].includes( activeTab ) ) {
+			return 'milliseconds';
+		}
+
+		return '';
+	};
+
+	return (
+		<div className="core-web-vitals-display">
+			<MetricTabBar activeTab={ activeTab } setActiveTab={ setActiveTab } { ...props } />
+			<div className="core-web-vitals-display__details">
+				<div className="core-web-vitals-display__description">
+					<span className="core-web-vitals-display__description-subheading">
+						{ translate( "Your site's %(displayName)s %(valuation)s", {
+							args: {
+								displayName: displayName.toLowerCase(),
+								valuation: displayValuation( valuation ),
+							},
+						} ) }
```

Comment:
tbh I think we would still have trouble even passing translated text as arguments, because we don't know how this phrase should be constructed in each language. 

As painful as it may sound I think we would need to create a dictionary with all variants so they could be translated as phrases in each language.

Reply:
Yeah I was thinking about the same, and while working on it it seemed to be more complex than using the translate templates. Now that it's mostly functional it seems much simpler to just create a dictionary for all strings, and it also simplifies translation. I moved most strings in https://github.com/Automattic/wp-calypso/pull/93411/commits/29f95b09e425e41053145bd30207ee7cc561c8b7